src/SingleBookingPage/bookingItem/Header.js

```diff
@@ -59,6 +60,9 @@ const Header = (props: Props) => {
                 onClick={e => {
                   e.preventDefault();
                   onDisplayAll();
+                  simpleTracker('smartFAQBookingOverview', {
```

Comment:
will this work properly? definition of simpleTracker: 

```
simpleTracker = (eventName: LogEvent, payload?: EventPayload) => () => {
```

a function that returns a function.